apps/compound/package.json

```diff
@@ -20,7 +20,10 @@
   "scripts": {
     "start": "dotenv -e .env -- react-app-rewired start",
     "build": "dotenv -e .env -- react-app-rewired build",
-    "test": "react-app-rewired test --passWithNoTests"
+    "test": "react-app-rewired test --passWithNoTests --forceExit",
```

Comment:
question: should we add this `--forceExit` flag to all safe apps `test` script?

Reply:
I will remove it, was debugging something